python/ray/dataframe/test/test_dataframe.py

```diff
@@ -108,6 +110,21 @@ def test_transpose(ray_df, pandas_df):
     assert(ray_df_equals_pandas(ray_df.transpose(), pandas_df.transpose()))
 
 
+@pytest.fixture
+def test_get(ray_df, pandas_df, key, default=None):
```

Comment:
I prefer this test to be as follows:
```python
def test_get(ray_df, pandas_df, key):
    assert ray_df.get(key).equals(pandas_df.get(key))
    assert ray_df.get(key, default='default').equals(pandas_df.get(key, default='default'))
```
This way we can get a test with and without default but don't have to run multiple tests.